src/config/users-and-groups.md

```diff
@@ -1,6 +1,37 @@
-# Users and groups
+# Users and Groups
 
-## Default groups
+The [useradd(8)](https://man.voidlinux.org/useradd.8),
+[userdel(8)](https://man.voidlinux.org/userdel.8) and
+[usermod(8)](https://man.voidlinux.org/usermod.8) commands are used to add,
+delete and modify users respectively. The
+[passwd(1)](https://man.voidlinux.org/passwd.1) command is used to change
+passwords.
+
+The [groupadd(8)](https://man.voidlinux.org/groupadd.8),
+[groupdel(8)](https://man.voidlinux.org/groupdel.8) and
+[groupmod(8)](https://man.voidlinux.org/groupmod.8) commands are used to add,
+delete and modify groups respectively. The
+[groups(1)](https://man.voidlinux.org/groups.1) command lists all groups a user
+belongs to.
+
+## sudo
+
+> Note: [sudo(8)](https://man.voidlinux.org/sudo.8) is installed by default but
+> may not be configured. It is only necessary to configure sudo if you wish to
+> use it.
+
+Use [visudo(8)](https://man.voidlinux.org/visudo.8) (as root) to edit the sudo
+configuration file.
```

Comment:
Since there's more than one config file related to `sudo` (`/etc/sudo.conf` as well as `/etc/sudoers`), i'd prefer this say:

"Use \[visudo(8)\](https://man.voidlinux.org/visudo.8) as root to edit the \`/etc/sudoers\` file."

Reply:
how about "use [visudo(8)] (as root) to edit the [sudoers(5)] file.", since that file has a manpage?

Reply:
You're right, that's better. :-) But my preference is to not have parentheses around "as root".

Reply:
sure thing